provider/dpdk_utils.py

```diff
@@ -0,0 +1,205 @@
+import subprocess
```

Comment:
```suggestion
from avocado.utils import process
```

Reply:
For the vDPA feature, some cases need to be run on the host. therefore, it's preferable for the dpdk_utils.py file to be usable independently, making it unsuitable to use avocado.utils.process.

I have updated, please help review. thanks.